packages/myst-cli/src/process/notebook.ts

```diff
@@ -143,5 +177,8 @@ export async function processNotebook(
     Promise.resolve([] as GenericNode[]),
   );
 
-  return { type: 'root', children: items };
+  logMessagesFromVFile(session, vfile);
```

Comment:
Actually, let's log these errors; we do care about validation after we've filtered the keys.